decidim-core/spec/types/static_page_topic_type_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "decidim/api/test"
+
+module Decidim
+  module Core
+    describe StaticPageTopicType do
+      let!(:model) { create(:static_page_topic, show_in_footer: true) }
+
+      include_context "with a graphql class type"
+      include_examples "timestamps interface"
+
+      describe "id" do
+        let(:query) { "{ id }" }
+
+        it "returns the static page's id" do
```

Comment:
It's better to just skip the resource. For instance here you're referring to "static page" but it should be "static page topic". Using the possessive (`'s`) also doesn't sound well in some fields (like `showInFooter`) 

```suggestion
        it "returns the id field" do
```